converters/toZigbee.js

```diff
@@ -4849,6 +4849,63 @@ const converters = {
             await entity.read('schneiderSpecificPilotMode', ['pilotMode'], {manufacturerCode: 0x105e});
         },
     },
+    schneider_temperature_measured_value: {
+        key: ['temperature_measured_value'],
```

Comment:
This is the temperature that the thermostat will try to achieve right? I think `target_temperature` is  a better name.

Reply:
This is the measured temperature, provided by an external sensor. 
The target temp is occupied_heating_setpoint